includes/Dashboard.php

```diff
@@ -281,14 +281,16 @@ public function enqueue_assets( $hook_suffix ) {
 			WEBSTORIES_VERSION
 		);
 
-		$this->enqueue_script( self::SCRIPT_HANDLE, [ Tracking::SCRIPT_HANDLE ] );
+		$all_scripts = $this->enqueue_script( self::SCRIPT_HANDLE, [ Tracking::SCRIPT_HANDLE ] );
 		$this->enqueue_style( self::SCRIPT_HANDLE, [ 'google-fonts' ] );
 
-		wp_localize_script(
-			self::SCRIPT_HANDLE,
-			'webStoriesDashboardSettings',
-			$this->get_dashboard_settings()
-		);
+		foreach ( $all_scripts as $script ) {
+			wp_localize_script(
+				$script,
+				'webStoriesDashboardSettings',
+				$this->get_dashboard_settings()
+			);
+		}
```

Comment:
That's incorrect.

Only the main script `self::SCRIPT_HANDLE` needs `wp_localize_script()` here.

Thus, we don't need `$all_scripts` here. `enqueue_script()` should return `void`.

Reply:
Happy to remove and agree not needed for `vendors~*.js`, but it's possible that webpack creates a `edit-story~stories-dashboard` chunk in the future. 

That is, unless we strictly want to disallow editor/dashboard code sharing by policy (do we have such any preventative checks yet?). I know you recently detangled some editor/dashboard code with async imports.

Reply:
Only the main entry point for the app (i.e. `edit/story.index.js`) needs the data passed by `wp_localize_script()`. Regardless of whether there is a `edit-story~stories-dashboard` chunk or not.

Reply:
Ah ok, I thought it did a transpile (e.g. rewrite macro) or similar.